std_msgs/Header header

geometry_msgs/PoseStamped[] position
geometry_msgs/PoseStamped[] velocity
geometry_msgs/PoseStamped[] acceleration
geometry_msgs/PoseStamped[] jerk

float32 yaw
float32 yaw_rate
uint32 current_seq
